autoload/maktaba/json/python.vim

```diff
@@ -0,0 +1,29 @@
+if !exists('s:disable_python')
+  let s:disable_python = 0
+endif
+
+
+""
+" @private
```

Comment:
`@public`? Not that it will be widely used, but it's made to be a hook usable from outside maktaba itself.